clients/coredata/event_test.go

```diff
@@ -23,12 +23,12 @@ import (
 	"net/http"
 	"net/http/httptest"
 	"testing"
-	"time"
+
+	"github.com/ugorji/go/codec"
```

Comment:
Move to below github.com/edgexfoundry imports (and separate from them with a blank line).